src/verify_getter.rs

```diff
@@ -89,14 +93,23 @@ fn find_or_insert_key_value_with_key(
     result
 }
 
+fn template_empty() -> SyntaxNode {
+    let ast = rnix::Root::parse(EMPTY_TEMPLATE);
+    let errors = ast.errors();
+    if errors.len() > 0 {
+        panic!("template_empty had an error: {:#?}", errors)
+    }
+    ast.syntax().first_child().unwrap().clone_for_update()
+}
```

Comment:
Maybe useless suggestion given the lifetime of a nix editor invocation, but would it make sense to have EMPTY_TEMPLATE_NODE that we just call `clone_for_update()` on?

Reply:
I think it's okay, because we are only going to ever reach these codepaths once per session: the first time when it is empty or non-exsistent. Also, empty vs non-existent are mutually exclusive so we will really only parse EMPTY_TEMPLATE once.